Documentation/Configuration/BackendEntryPoint/Index.rst

```diff
@@ -0,0 +1,188 @@
+
+..  note::
+    The install tool is still available via :samp:`/typo3/install.php`.
+
+..  _WSTG-CONF-05: https://owasp.org/www-project-web-security-testing-guide/v42/4-Web_Application_Security_Testing/02-Configuration_and_Deployment_Management_Testing/05-Enumerate_Infrastructure_and_Application_Admin_Interfaces
+
+
+..  _backend-entry-point-configuration:
+
+Configuration
+=============
+
+The configuration can be done in the backend via
+:guilabel:`Admin Tools > Settings > Configure Installation-Wide Options`:
+
+..  figure:: /Images/ManualScreenshots/AdminTools/ConfigureEntryPoint.png
+    :alt: Configure the entry point via GUI
+    :class: with-shadow
+
+    Configure the entry point via GUI
+
+or manually in :ref:`config/system/settings.php <typo3ConfVars-settings>` or
+:ref:`config/system/additional.php <typo3ConfVars-additional>`.
+
+..  _backend-entry-point-specific-path:
+
+Configure a specific path
+-------------------------
+
+..  code-block:: php
+    :caption: config/system/additional.php
+
+    $GLOBALS['TYPO3_CONF_VARS']['BE']['entryPoint'] = '/my-specific-path';
+
+Now point your browser to :samp:`https://example.org/my-specific-path` to
+log into the TYPO3 backend.
+
+
+..  _backend-entry-point-specific-subdomain:
+
+Use a distinct subdomain
+------------------------
+
+..  code-block:: php
+    :caption: config/system/additional.php
+
+    $GLOBALS['TYPO3_CONF_VARS']['BE']['entryPoint'] = 'https://my-backend-subdomain.example.org';
+    $GLOBALS['TYPO3_CONF_VARS']['SYS']['cookieDomain'] = '.example.org';
+
+Now point your browser to `https://my-backend-subdomain.example.org/` to log
+into the TYPO3 backend.
+
+Note that the :php:`$GLOBALS['TYPO3_CONF_VARS']['SYS']['cookieDomain']` is
+necessary, so that backend users can preview website pages or use the admin
+panel.
+
+..  _backend-entry-point-legacy-free:
+
+Legacy-free installation
+------------------------
+
+The legacy entry point `/typo3/index.php` is no longer needed and deprecated in
+favor of handling all backend and frontend requests with :samp:`/index.php`. The
+entry point is still in place in case webserver configuration has not been
+adapted yet and the maintenance and emergency tool is still available via
+:samp:`/typo3/install.php` in order to work in edge cases like broken web server
+routing.
+
+In Composer mode is an additional configuration option for the deactivation of
+the legacy entry point which can be defined in the project's
+:file:`composer.json`.
+
+..  code-block:: json
+    :caption: /path/to/project/composer.json (Excerpt)
+
+    {
+        "extra": {
+            "typo3/cms": {
+                "install-deprecated-typo3-index-php": false
+            }
+        }
+    }
+
+..  _backend-entry-point-migration:
+
+Migration
+=========
+
+A silent update is in place which automatically updates the webserver
+configuration file when accessing the install tool, at least for Apache and
+Microsoft IIS webservers.
```

Comment:
Maybe for people to understand the most common place for this, we could mention `.htaccess` specifically:

```
at least for Apache (:file:`.htaccess`) and
Microsoft IIS webservers (:file:`web.config`).
```

I guess you want to express that configuration could also be done in the central webserver configuration (like for nginx), but in this case TYPO3 can only automatically adapt the specific htaccess/web.config files, so IMO it would be okay to explicitly mention these file names?

Reply:
Done